src/vec_buffers.h

```diff
@@ -153,10 +153,89 @@ class Buffers final {
         unsigned data_len,
         unsigned group_len);
     void radix2_fft_inv_prepare(const Buffers<T>& input);
+    void reset_meta();
     friend bool operator==<T>(const Buffers<T>& lhs, const Buffers<T>& rhs);
     void dump(void);
     void swap(unsigned i, unsigned j);
 
+    /** Calculate meta size given a buffer size
+     *  meta size := ceil(size * sizeof(T) / CHAR_BIT)
+     *
+     * @param s - given size, in words
+     * @return meta size, in bytes
+     */
+    static size_t compute_meta_size(size_t s)
+    {
+        assert(s > 0);
+
+        const size_t bytes = s * sizeof(T);
+        size_t r = bytes / CHAR_BIT;
+        while (r * CHAR_BIT < bytes) {
+            r++;
+        }
+        return r;
+    };
+
+    /** Calculate buffer size given a meta_size
+     *  buffer size := ceil(meta_size * CHAR_BIT / sizeof(T))
+     *
+     * @param m_size - given meta size, in bytes
+     * @return meta size
+     */
+    static size_t compute_size(size_t m_size)
+    {
+        assert(m_size > 0);
+
+        const size_t bytes = m_size * CHAR_BIT;
+        size_t r = bytes / sizeof(T);
+        while (r * sizeof(T) < bytes) {
+            r++;
+        }
+        return r;
+    };
+
+    /** Calculate conventional size of buffers
+     * Conentional size is the lowest number of words that is at least a given
```

Comment:
s/Conentional/conventional/

Also, skip a line between the short description (the first line) and the detailed one (the rest).